Mathlib/RingTheory/Localization/Module.lean

```diff
@@ -198,4 +258,27 @@ def LinearMap.extendScalarsOfIsLocalization (f : M →ₗ[R] N) : M →ₗ[A] N
 @[simp] lemma LinearMap.extendScalarsOfIsLocalization_apply (f : M →ₗ[A] N) :
     f.extendScalarsOfIsLocalization S A = f := rfl
 
-end
+variable {M' : Type*} [AddCommMonoid M'] [Module R M'] [Module A M']
+  [IsScalarTower R A M'] (f : M →ₗ[R] M')  [IsLocalizedModule S f]
+
+/-- If `(f : M →ₗ[R] M')` is a localization of modules, then the map
+`(localization S) × M → N, (s, m) ↦ s • f m` is the tensor product (insomuch as it is the universal
+bilinear map).
+In particular, there is an isomorphism between `LocalizedModule S M` and `(Localization S) ⊗[R] M`
+given by `m/s ↦ (1/s) ⊗ₜ m`.
+-/
+theorem IsLocalizedModule.isBaseChange : IsBaseChange A f := by
```

Comment:
Side comment: it should be easy to show that for a submodule N of M, the restriction of `f` to `N → span A (N.map f)` also satisfies `IsLocalizedModule A`. Combined with this theorem it should be easy to show `A` is flat over `R`.

I think it's also true that `span A (N.map f)` is an OrderIso between R-submodules of M and A-submodules of M', with inverse given by `N'.comap f`.

Reply:
Thanks for the review!

> Side comment: it should be easy to show that for a submodule N of M, the restriction of f to N → span A (N.map f) also satisfies IsLocalizedModule A. Combined with this theorem it should be easy to show A is flat over R.

> I think it's also true that span A (N.map f) is an OrderIso between R-submodules of M and A-submodules of M', with inverse given by N'.comap f.

But that would be for another PR, right? 😅

Reply:
Sure, that's why it's a side comment ...